src/lib/groovy/modsdotgroovy/EntrypointsBuilder.groovy

```diff
@@ -32,6 +37,30 @@ class EntrypointsBuilder {
         propertyMissing(name, args)
     }
 
+    /**
+     * Adds a client entrypoint.
+     * @param args either the single value of the entrypoint or a list of values.
+     */
+    void client(args) {
+        propertyMissing(ModsDotGroovy.platform === Platform.QUILT ? 'client_init' : 'client', args)
+    }
+
+    /**
+     * Adds a server entrypoint.
+     * @param args either the single value of the entrypoint or a list of values.
+     */
+    void server(args) {
+        propertyMissing(ModsDotGroovy.platform === Platform.QUILT ? 'server_init' : 'server', args)
+    }
+
+    /**
+     * Adds a main entrypoint.
+     * @param args either the single value of the entrypoint or a list of values.
+     */
+    void main(args) {
+        propertyMissing(ModsDotGroovy.platform === Platform.QUILT ? 'init' : 'main', args)
+    }
```

Comment:
Please turn this into two setter methods to line up with the rest of the DSL and improve IDE support, for example:
```groovy
void setMain(String entrypoint)
void setMain(List<String> entrypoint)
```